packages/webidl/webidl.1.2/opam

```diff
@@ -11,6 +11,7 @@ depends: [
   "ocaml" {>= "4.03.0"}
   "ocamlfind" {build & >= "1.7.1"}
   "ocamlbuild" {build & >= "0.9.3"}
+  "menhir" {build & >= "20170607" & < "20211215"}
```

Comment:
are you sure `1.2` has a dependency on menhir?

http://check.ocamllabs.io/log/1648270349-9d91fc6519d4e3db3bee7883ac155eea5b3b54d3/4.14/good/webidl.1.2 doesn’t show any issues